inc/section_12_drupal_modules.inc

```diff
@@ -18,23 +20,18 @@ function section_12_drupal_modules() {
     'cust_modules' => [
       '#title' => t('Custom Modules'),
       '#description' => t('Ensure that custom modules are in a "custom" folder.'),
-      // 'handbook_page' => array(
-      // '#text' => t( 'Google Analytics module' ),
-      // '#url' => $google_anaytics_route,
     ],
     'coder_module' => [
       '#title' => t('Coder Module'),
       '#description' => t('Run the Coder module and fix any issues it discovers in the custom modules.'),
-      // 'handbook_page' => array(
-      // '#text' => t( 'Google Analytics module' ),
-      // '#url' => $google_anaytics_route,
+      'handbook_page' => [
+        '#text' => t('Coder module'),
+        '#url' => Url::fromUri('https://www.drupal.org/project/coder'),
+      ],
     ],
     'devel_module' => [
       '#title' => t('Devel Module'),
       '#description' => t('Ensure that the Devel module is disabled on the production site.'),
```

Comment:
You could like to the modules page.

Reply:
You could _link_ to the modules page.

Reply:
That was my original thought, but then I read "Coder is not a module, it is used as command line tool and in IDEs." on the drupal.org page